components/Button.tsx

```diff
@@ -10,6 +10,7 @@ export const buttonVariants = cva(
         primary: 'bg-blue text-neutral-100',
         secondary: 'bg-inherit',
         outline: 'bg-none shadow-[inset_0_0_0_2px_#0015CE] text-neutral-900',
+        ghost: 'w-full text-center h-[100px] relative top-40'
```

Comment:
Some of these should be for the specific button you are using it for, a variant should only give base styling like centered, background color, border or so on. 

With this button component you can still add a className to where you are using it and add all the extra styling you need.